### DISCUSSION TITLE
Paramaters

### DISCUSSION THREAD

**Discussion by P34R1** [ReadieFur/BSDP-Overlay#8] [Q&A] [answered]
How do you use the settings I can't figure it out.

**Comment by ReadieFur** [comment, marked as answer]
Here is an example http://u-readie.global-gaming.co/bsdp-overlay/?flipHori&scale=0.9

**Reply by P34R1** [reply]
Thank you for the example I forgot to say i don't need help after looking up how to use URL Parameters but anyway love the overlay.

**Reply by ReadieFur** [reply]
> Thank you for the example I forgot to say i don't need help after looking up how to use URL Parameters but anyway love the overlay.

Thanks for that :). Hopefully when I get some more time I can get around to getting the overlay editor out in an early public beta stage sometime in the near future if all goes well.
I've added back some of the missing features as well as adding colour customisation #6.
